src/Qir/Execution/CommandLineCompiler/CommandLineCompiler.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
To avoid confusion with the Q# command line compiler, I would rename this to CommandLineBuildTools.

Reply:
Renamed to QirCommandLineTool.